### PR TITLE
Task/hw 53616 ui automation new

### PR DESCRIPTION
**Pull request by malaw-hw**
it covers the following:-
- Create Transfer with FX
- Create Transfer without FX
- FX quote expired error 
- FX quote with no fee section

### CODE REVIEW

File: UITests/TransferFunds/TransferUserFundsConfirmationTest.swift [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+        transferFunds = TransferFunds(app: app)
+        transferFundsConfirmation = TransferFundsConfirmation(app: app)
+    }
+
+    override func tearDown() {
+        mockServer.tearDown()
+    }
+
+    func testTransferFundsConfirmation_noFX() {
```
Comment (jsaini-hw) [inline UITests/TransferFunds/TransferUserFundsConfirmationTest.swift]: rename to withoutFX instead of noFX

Comment (malaw-hw) [reply UITests/TransferFunds/TransferUserFundsConfirmationTest.swift]: ok

File: UITests/Helper/TransferFunds/TransferFundsConfirmation.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import Common
+import XCTest
+
+class TransferFundsConfirmation {
+    var app: XCUIApplication
+
+    // Destination Section
+    var addSelectDestinationLabel: XCUIElement
```
Comment (skoong) [inline UITests/Helper/TransferFunds/TransferFundsConfirmation.swift]: The naming of this is confusing, there shouldn't be any selection or add in this page.  You can rename it to transferDestinationLabel and transferDestinationDetail.

File: UITests/TransferFunds/TransferUserFundsConfirmationTest.swift [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+        XCTAssertTrue(transferFundsConfirmation.confirmButton.exists)
+        transferFundsConfirmation.confirmButton.tap()
+
+        // Assert go back to the menu page
+        waitForExistence(transferFundMenu)
+        XCTAssertTrue(transferFundMenu.exists)
+    }
+
+    func testTransferFundsConfirmation_timeOutError() {
```
Comment (skoong) [inline UITests/TransferFunds/TransferUserFundsConfirmationTest.swift]: Rename to testTransferFundsConfirmation_timeoutError

File: UITests/TransferFunds/TransferUserFundsConfirmationTest.swift [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+        XCTAssertEqual(app.cells.element(boundBy: 17)
+            .staticTexts["scheduleTransferSummaryTextValue"].label, "2.00")
+
+        XCTAssertEqual(app.cells.element(boundBy: 18)
+            .staticTexts["scheduleTransferSummaryTextLabel"].label, "You will receive:")
+        XCTAssertEqual(app.cells.element(boundBy: 18)
+            .staticTexts["scheduleTransferSummaryTextValue"].label, "5,855.17")
+
+        XCTAssertTrue(transferFundsConfirmation.noteLabel.exists)
+        XCTAssertEqual(transferFundsConfirmation.noteDescription.value as? String, "Transfer All")
```
Comment (skoong) [inline UITests/TransferFunds/TransferUserFundsConfirmationTest.swift]: Can you add a case where there is no notes and verify that the section is not displayed?

**Pull request merged**